bpf/process/pfilter.h

```diff
@@ -396,6 +396,10 @@ selector_process_filter(__u32 *f, __u32 index, struct execve_map_value *enter,
 	__u32 len;
 	__u64 i;
 
+	/* Do binary filter first for selector index */
+	if (!match_binaries(index))
```

Comment:
we might get more binary filters e.g. uid/gid so this might become something like process_filter_binary with multiple filters soon. But fine its ok for now.